collada/material.py

```diff
@@ -576,12 +579,13 @@ def load(collada, localscope, node):
         localscope = {} # we have our own scope, shadow it
         params = []
         id = node.get('id')
-        profilenode = node.find( tag('profile_COMMON') )
+        profilenode = node.find( collada.tag('profile_COMMON') )
         if profilenode is None:
-            raise DaeUnsupportedError('Found effect with profile other than profile_COMMON')
+            return
+            # raise DaeUnsupportedError('Found effect with profile other than profile_COMMON')
```

Comment:
Did you mean to comment out this exception?

Reply:
Good catch, fixed in https://github.com/pycollada/pycollada/pull/86/commits/c2465925b574802b84599f44b23c64161d4c6a38